sdk/storage/azblob/appendblob/client_test.go

```diff
@@ -28,8 +29,16 @@ import (
 )
 
 func Test(t *testing.T) {
-	suite.Run(t, &AppendBlobRecordedTestsSuite{})
-	//suite.Run(t, &AppendBlobUnrecordedTestsSuite{})
+	recordMode := recording.GetRecordMode()
+	t.Logf("Running appendblob Tests in %s mode\n", recordMode)
+	if recordMode == "live" {
+		suite.Run(t, &AppendBlobRecordedTestsSuite{})
+		suite.Run(t, &AppendBlobUnrecordedTestsSuite{})
+	} else if recordMode == "playback" {
+		suite.Run(t, &AppendBlobRecordedTestsSuite{})
+	} else if recordMode == "record" {
```

Comment:
```suggestion
	if recordMode == recording.LiveMode {
		suite.Run(t, &AppendBlobRecordedTestsSuite{})
		suite.Run(t, &AppendBlobUnrecordedTestsSuite{})
	} else if recordMode == recording.PlaybackMode {
		suite.Run(t, &AppendBlobRecordedTestsSuite{})
	} else if recordMode == recording.RecordMode {
```

Reply:
Fixed (I saw that `recording.GetRecordMode()` returns a string and assumed there weren't constants for this)